pkg/agent/agent_windows.go

```diff
@@ -173,3 +174,19 @@ func (i *Initializer) initHostNetworkFlows() error {
 	}
 	return nil
 }
+
+// setupTunnelInterfaceOptions sets local_ip option of tunnel interface.
+// On windows, the local_ip option is needed to be configured to help OVS set right source IP
+// for encap packets.
+func (i *Initializer) setupTunnelInterfaceOptions(ifName string) error {
+	options, err := i.ovsBridgeClient.GetInterfaceOptions(ifName)
```

Comment:
I do not feel this is needed, as Agent manages the interface and knows all options on it. If you change to what I suggested above, then after tunnel interface is recovered at Agent startup, you can check all options including the IP, and update them if needed.

Reply:
`setupTunnelInterfaceOptions` is for provide different behavior on windows and linux. I will consider how to change it after reslove the above option issue.